### PR TITLE
Added Total.js v4 and Total.js v5 framework.

### PR DESCRIPTION
**Pull request by petersirka**

### CODE REVIEW

**Review by waghanza — CHANGES_REQUESTED** [review]
Thanks for your contribution :tada:, but maybe we can only include the latest version of one framework.

I understand that having multiple versions could help developers to enhance their framework, but having more than one version of each framework could complexify results.

Could you choose one version? Rather the latest, or even better the lts one, if such approach exist for totaljs

**Comment by petersirka** [comment]
Thank you. Please keep only `Total5`.

**Comment by waghanza** [comment]
Could you please proceed ?

I cannot push in PR from master

**Review by waghanza — CHANGES_REQUESTED** [review]
Could you please rename it in totaljs ?

File: javascript/total5/app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require('total5')
+
+ROUTE('GET /', function($) {
+  $.text('Hello world!');
```
Comment (waghanza) [inline javascript/total5/app.js:4]:
```suggestion
  $.text('');
```

File: javascript/total5/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: totaljs.com
+  version: 0.0.6
```
Comment (waghanza) [inline javascript/total5/config.yaml:3]:
```suggestion
  version: 0.0
```

File: javascript/totaljs/config.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: totaljs.com
+  version: 0.0.6
```
Comment (waghanza) [inline javascript/totaljs/config.yaml:3]:
```suggestion
  version: 0.0
```

File: javascript/totaljs/app.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+require('total5')
+
+ROUTE('GET /', function($) {
+  $.text('Hello world!');
```
Comment (waghanza) [inline javascript/totaljs/app.js:4]:
```suggestion
  $.text('');
```

**Review by waghanza — APPROVED** [review]

**Review by waghanza — CHANGES_REQUESTED** [review]
A route is not correct, I have made a suggestion

**Review by waghanza — APPROVED** [review]

**Pull request merged**